api/app/tests/fireweather_bot/test_hourly_actuals.py

```diff
@@ -44,3 +93,68 @@ def mock_process_csv(self, filename: str):
     assert excinfo.value.code == os.EX_SOFTWARE
     # Assert that rocket chat was called.
     assert rocket_chat_spy.call_count == 1
+
+
+def test_parse_hourly_actual():
+    """ Valid fields are set when values exist """
+    weather_reading = WeatherReading(
+        datetime=get_utc_now(),
+        temperature=0.0,
+        relative_humidity=0.0,
+        wind_speed=0.0,
+        wind_direction=0.0,
+        barometric_pressure=0.0,
+        precipitation=0.0,
+        dewpoint=0.0,
+        ffmc=0.0,
+        isi=0.0,
+        fwi=0.0
+    )
+
+    hourly_actual = hourly_actuals.parse_hourly_actual(1, weather_reading)
+    assert hourly_actual.rh_valid is True
+    assert hourly_actual.temp_valid is True
+    assert hourly_actual.wdir_valid is True
+    assert hourly_actual.precip_valid is True
+    assert hourly_actual.wspeed_valid is True
```

Comment:
I don't believe these assertions of the `_valid` flags are relevant anymore?

Reply:
It's possible these contradict with wf1's observation_valid_ind unless we really trust their output

Reply:
my understanding was that we will no longer have _valid flags for each individual wx variable anymore?